proto/beacon/p2p/v1/messages.proto

```diff
@@ -35,4 +35,18 @@ message ENRForkID {
 message MetaData {
   uint64 seq_number = 1;
   bytes attnets = 2 [(ethereum.eth.ext.ssz_size) = "8", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitvector64"];
+}
+
+/*
+ Spec Definition:
+ MetaData
+ (
+ seq_number: uint64
+ attnets: Bitvector[ATTESTATION_SUBNET_COUNT]
+ )
+*/
+message MetaDataV2 {
```

Comment:
This is not related to this PR, at least by looking at the title and the description

Reply:
done